### DISCUSSION TITLE
Minor styling issues

### DISCUSSION THREAD

**Discussion by 5E-324** [danny-avila/LibreChat#4538] [Feature Requests & Suggestions]
### What happened?

Chat page:
- Some tooltips have transparent background.
- Some margin is missing in bookmarks popup. (Note: this would be fine if no adjacent elements are highlighted at the same time, but here multiple bookmarks may be selected simultaneously)
- Indicator of multi-conversation has inconsistent rounding.
- On right sidebar, if a section is expanded, the corresponding icon will have wrong color when the sidebar is contracted

Settings:
- Toggle switches are slightly off center and are misaligned with text.
- In speech tab, some margin is missing.

### Steps to Reproduce

Install with docker, open in browser.

### What browsers are you seeing the problem on?

Firefox, Chrome

### Screenshots

Tooltips with transparent background:

<img width="351" alt="tooltip1" src="https://github.com/user-attachments/assets/d18d195e-4618-4fd4-b50b-892ad4877783">
<img width="272" alt="tooltip2" src="https://github.com/user-attachments/assets/e9b29ca1-c8bf-4a19-98a7-0e1c79dfe3b6">
<img width="300" alt="tooltip3" src="https://github.com/user-attachments/assets/f79b0cc9-52e9-41e8-a9e1-0d21555fd02e">

Bookmark popup:

<img width="342" alt="bookmarks1" src="https://github.com/user-attachments/assets/7ad6cede-621f-48b8-a79d-3d63a462fda9">
<img width="221" alt="bookmarks2" src="https://github.com/user-attachments/assets/f9e4e06b-0988-47e7-9a95-3b840b210c78">

Inconsistent rounding:

<img width="596" alt="rounding" src="https://github.com/user-attachments/assets/c2c764f1-eff0-475c-aff0-36a1c4fb554e">

Right sidebar:
No section expanded:

<img width="291" alt="sidebar1a" src="https://github.com/user-attachments/assets/c233a2ad-0a7c-49ac-a7cf-176e22f7c26e">
<img width="51" alt="sidebar1b" src="https://github.com/user-attachments/assets/2173b0cd-b1c4-4751-a633-eb04e53f009f">

Prompts expanded (same for other sections):

<img width="289" alt="sidebar2a" src="https://github.com/user-attachments/assets/2bdb6df4-767e-498e-93ca-5506abc63fd3">
<img width="48" alt="sidebar2b" src="https://github.com/user-attachments/assets/1649e02c-87b2-4d35-86e3-22ef44eac23b">

Misaligned toggle switches:

<img width="537" alt="toggle1" src="https://github.com/user-attachments/assets/c37cc74c-ed88-4026-8497-5ff871d6ea0a">

This also exist in other tabs. Some are less obvious:

<img width="535" alt="toggle2" src="https://github.com/user-attachments/assets/e3824c84-0721-44be-8be4-bd289f5639b0">

Missing margin:

<img width="527" alt="margin" src="https://github.com/user-attachments/assets/fb4b2921-60e7-4e93-843d-55dfce88cb3d">

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by KiGamji** [comment]
the New Chat button is also missing on mobile version, leaving a span after where it should be

**Comment by KiGamji** [comment]
there are some more issues btw. when there is an error in a red box, and you send another message, after it finishes the completion, the rest of conversation just folds under another branch
